CMakeLists.txt

```diff
@@ -227,6 +227,46 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
           "-gencode arch=compute_90a,code=sm_90a")
   endif()
 
+  #
+  # For the Machete kernels we automatically generate sources for various 
+  # preselected input type pairs and schedules.
+  # Generate sources:
+  execute_process(
```

Comment:
Does this regenerate all the files every time you build?

Reply:
ya it does, really hasnt be an issue since ccache handles it, but I can look at something fancier than this